.github/workflows/android-ci.yml

```diff
@@ -9,6 +9,7 @@ on:
   pull_request:
     branches:
       - main
+  workflow_dispatch:
```

Comment:
Has this line any sense? Or you left it by accident after some local experiments?

Reply:
Not important anymore, but it's very useful if you want to trigger a CI run for a branch without an open PR. We could add it to all workflows if you like, or just get rid of it here.